docker-compose-conflictmonitor.yml

```diff
@@ -0,0 +1,176 @@
+services:
+  conflictmonitor:
+    profiles:
+      - conflictmonitor
+    # -------- To build the image locally --------
+    # image: jpo-conflictmonitor:latest
+    # build: ./services/intersection-api/jpo-conflictmonitor
+    #
+    # -------- To use a remote image -------------
+    image: usdotjpoode/jpo-conflictmonitor:latest
+    # --------------------------------------------
+    restart: ${RESTART_POLICY}
+    ports:
+      - '8082:8082'
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP:?error}
+      KAFKA_BOOTSTRAP_SERVERS: ${KAFKA_BOOTSTRAP_SERVERS:?error}
+    healthcheck:
+      test: ['CMD', 'java', '-version']
+      interval: 10s
+      timeout: 10s
+      retries: 20
+    logging:
+      options:
+        max-size: '10m'
+        max-file: '5'
+    deploy:
+      resources:
+        limits:
+          memory: 3G
+    depends_on:
+      kafka:
+        condition: service_healthy
+        required: false
+
+  ode:
+    profiles:
+      - conflictmonitor
+    # -------- To build the image locally --------
+    # image: jpo-ode:latest
+    # build: ./services/intersection-api/jpo-conflictmonitor/jpo-geojsonconverter/jpo-ode
```

Comment:
Question (non-blocking): Are the jpo-geojsonconverter and jpo-ode repositories still being used as Git submodules? I thought we had transitioned to using Maven and DockerHub instead.

Reply:
Good point - these never got removed. Took out the local build documentation as well, as that is no longer directly supported